views/pages/organisation/user.html

```diff
@@ -16,13 +17,23 @@
     <div class="govuk-grid-column-full">
         {% block mainContent %}
         <div class="govuk-grid-row">
+            {% if updatedUser %}
+                {% set html %}
+                <h2>You have successfully edited {{ updatedUser }}'s details</h2>
```

Comment:
I'd probably say updated, rather than edited. Potentially could say updated team or name depending on what it is but that's probably a lot more work and not required.

Reply:
Agree with the wording change to "updated". Don't need to go into specifics at this point about _what_ was updated.

Reply:
made changes to wording